### DISCUSSION TITLE
增加按照页面Xpath进行筛选进行翻译

### DISCUSSION THREAD

**Discussion by zhangshuohua-nuaa** [immersive-translate/immersive-translate#961] [Ideas]
### 新功能 | New Feature

1. Xpath语法选择对应元素
2. 进行特定区域翻译

### 为什么您想要添加这些功能 | Why you want to add these features

1. 全局翻译的时候，某些元素会出现遮挡，添加Xpath语法选择对应的翻译区域
2. 翻译内容多且复杂的时候，会出现不希望翻译的部分也被翻译

### 提议 | Proposal

允许用户自定义特定页面的Xpath语法，翻译的时候获取指定元素进行翻译

**Comment by keyboard3** [comment]
支持通过自定义规则配置选择器指定翻译元素。 https://immersivetranslate.com/docs/advanced/
